openadapt/config.py

```diff
@@ -110,6 +114,37 @@ def getenv_fallback(var_name):
     return rval
 
 
+def persist_env(var_name, val):
+    """
+    The `persist_env` function is used to persist environment variables by writing them to a `.env`
+    file.
+
+    :param var_name: The `var_name` parameter is a string that represents the name of the environment
+    variable you want to persist or update in the `.env` file
+    :param val: The `val` parameter in the `persist_env` function is the value that you want to assign
+    to the environment variable specified by `var_name`
```

Comment:
Thanks @0dm !

Can you please modify the documentation to follow the google python style guide?

Also please add `env_file_path=ENV_FILE_PATH` as a parameter.

Also please add type annotations.

Reply:
[aacf116](https://github.com/OpenAdaptAI/OpenAdapt/pull/375/commits/aacf11616f9010b07dffa311045326dad3ac1415)